src/profile/ResetPassword.js

```diff
@@ -6,7 +6,8 @@ import FormGroup from "react-bootstrap/lib/FormGroup";
 import ControlLabel from "react-bootstrap/lib/ControlLabel";
 import FormControl from "react-bootstrap/lib/FormControl";
 import HelpBlock from "react-bootstrap/lib/HelpBlock";
-import Glyphicon from "react-bootstrap/lib/Glyphicon";
+// import Glyphicon from "react-bootstrap/lib/Glyphicon";
```

Comment:
As an FYI, in the future you can delete this line :) If we feel that we should leave it commented in instead, we will note it in the review!

Reply:
Ok, sounds good!